### PR TITLE
Increase connection establish timeout and fix its error object

### PR DESCRIPTION
**Pull request by pevogam**
Signed-off-by: Plamen Dimitrov <plamen.dimitrov@intra2net.com>

### CODE REVIEW

File: avocado/core/nrunner/task.py [hunk 1]
```diff
@@ -47,15 +47,15 @@ def post(self, status):
             host, port = self.uri.split(":")
             port = int(port)
             if self.connection is None:
-                for _ in range(30):
+                for _ in range(600):
```
Comment (richtja) [inline avocado/core/nrunner/task.py:50]: Hi @pevogam can you please give me more context to this change? I don't understand why you need to increase the timeout. Thank you.

Comment (pevogam) [reply avocado/core/nrunner/task.py:50]: Hi @richtja at least empirically most containers as well as remote hosts could take much longer than 30 seconds in various situations (hardware load, being booted, etc.). As the above loop will be interrupted earlier if the connection is established earlier it won't slow down faster cases but will give more tolerance to slower cases (I have obtained multiple timeouts here due to 30 seconds being too restrictive).

**Review by richtja — APPROVED** [review]
Hi @pevogam, LGTM, thanks.

File: avocado/core/nrunner/task.py (hunk 1 above)
Comment (richtja) [reply avocado/core/nrunner/task.py:50]: I see, thanks for the explanation.

**Pull request merged**